### PR TITLE
✨ Build 'Add Class' component

### PR DESCRIPTION
**Pull request by fabiglahn**
## Description

For the different days, there's a component with the different classes that can be expanded using the array. If there's time, I'll change it to another database though...

## Design

Bildschirmfoto 2021-12-10 um 15 04 50

### CODE REVIEW

File: src/Components/AddClass/AddClass.tsx [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+];
+
+export default function AddClass() {
+  return (
+    <>
+      {classes.map(({ name, time, place }) => (
+        <article className={styles.singlecard} key={name}>
+          <h4 className={styles.coursename}>{name}</h4>
+          <div className={styles.withbutton}>
+            <div className={styles.allinfo}>
```
Comment (lajanzen) [inline src/Components/AddClass/AddClass.tsx]: I'd use camel case for your class names for better readability.
```suggestion
        <article className={styles.singleCard} key={name}>
          <h4 className={styles.courseName}>{name}</h4>
          <div className={styles.withButton}>
            <div className={styles.allInfo}>
```

File: src/Components/AddClass/AddClass.tsx [hunk 2]
```diff
@@ -0,0 +1,38 @@
...
+  { name: "pilates & tone", time: " 21:30", place: " Studio" },
+];
+
+export default function AddClass() {
+  return (
+    <>
+      {classes.map(({ name, time, place }) => (
+        <article className={styles.singlecard} key={name}>
+          <h4 className={styles.coursename}>{name}</h4>
+          <div className={styles.withbutton}>
```
Comment (lajanzen) [inline src/Components/AddClass/AddClass.tsx]: Not sure if "withButton" is a good name here - I'd consider renaming it ;)

File: src/Components/AddClass/assets/clock.svg [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<svg
```
Comment (lajanzen) [inline src/Components/AddClass/assets/clock.svg:1]: You could consider making your svgs components, that way you could import them as such, too :)

File: src/Components/AddClass/assets/clock.svg [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+          width="16"
+          height="16"
+          viewBox="0 0 16 16"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/AddClass/assets/clock.svg:6]:
```suggestion
```

File: src/Components/AddClass/assets/location.svg [hunk 5]
```diff
@@ -0,0 +1,16 @@
+<svg
+          width="14"
+          height="19"
+          viewBox="0 0 14 19"
+          fill="none"
+          xmlns="http://www.w3.org/2000/svg"
```
Comment (lajanzen) [inline src/Components/AddClass/assets/location.svg:6]:
```suggestion
```

File: src/Components/AddClass/assets/plus.svg [hunk 6]
```diff
@@ -0,0 +1,4 @@
+<svg width="60" height="60" viewBox="0 0 60 60" fill="none" xmlns="http://www.w3.org/2000/svg">
```
Comment (lajanzen) [inline src/Components/AddClass/assets/plus.svg:1]:
```suggestion
<svg width="60" height="60" viewBox="0 0 60 60" fill="none">
```

File: src/Components/AddClass/AddClass.module.css [hunk 7]
```diff
@@ -0,0 +1,32 @@
...
+}
+
+.info {
+  color: var(--color-text-main);
+  font-size: 1.25rem;
+}
+
+.allinfo {
+  display: grid;
+  grid-template-rows: 1fr 1fr;
```
Comment (lajanzen) [inline src/Components/AddClass/AddClass.module.css]:
```suggestion
```

**Review by lajanzen — APPROVED** [review]

File: src/Components/AddClass/assets/plus.svg (hunk 6 above)
Comment (fabiglahn) [reply src/Components/AddClass/assets/plus.svg:1]: Wenn ich das rausnehme, dann wird mir das SVG nicht mehr angezeigt?!

**Pull request merged**